transport/src/androidMain/kotlin/com/genesys/cloud/messenger/transport/core/MobileMessenger.kt

```diff
@@ -8,8 +8,17 @@ import com.genesys.cloud.messenger.transport.shyrka.receive.DeploymentConfig
 import com.genesys.cloud.messenger.transport.util.logs.Log
 import com.genesys.cloud.messenger.transport.util.logs.LogTag
 
+/**
+ * Transport entry point.
+ */
 object MobileMessenger {
 
+    /**
+     * Creates an instance of [MessagingClient] based on the provided configurations.
+     *
+     * @param context is an application context.
+     * @param configuration to rely on when setup the messenger.
```

Comment:
* @param context an application context.
* @param configuration the configuration parameters for setting up the client.